infra/feast-operator/internal/controller/services/services.go

```diff
@@ -135,6 +160,24 @@ func (feast *FeastServices) createDeployment(feastType FeastServiceType) error {
 	return nil
 }
 
+func (feast *FeastServices) createPVC(pvcCreate *feastdevv1alpha1.PvcCreate, feastType FeastServiceType) error {
+	logger := log.FromContext(feast.Context)
+	pvc := feast.initPVC(feastType)
+	err := feast.setPVC(pvc, pvcCreate, feastType)
+	if err != nil {
+		return err
+	}
+	if op, err := controllerutil.CreateOrUpdate(feast.Context, feast.Client, pvc, controllerutil.MutateFn(func() error {
+		return nil
```

Comment:
```suggestion
	if op, err := controllerutil.CreateOrUpdate(feast.Context, feast.Client, pvc, controllerutil.MutateFn(func() error {
		return feast.setPVC(pvc, pvcCreate, feastType)
```

Reply:
the PVC object mods need to happen within the `CreateOrUpdate func ()`

Reply:
was this moved because of the immutable nature of the PVC?

Reply:
if so, we could be explicit with PVC creation and use `feast.Client.Create()` here instead?

Reply:
Tried the Create option but creation was not an issue. Deletion seems to fail, at least in the tests (just restored them to see the failure)